src/post/workshed_utils.ts

```diff
@@ -67,6 +67,26 @@ export function getBestCycle(): TrainSet.Cycle {
   return [...trainCycle];
 }
 
+export function getBestOffset(): number {
```

Comment:
We should probably cache this - I doubt the offset would change over the course of a run